### DISCUSSION TITLE
【希望能给一些方向性的建议】UMI@3.5.39 本地开发正常，build打包后部署服务器上无法打开页面，页面白屏，没报错，model里的路由监听也不生效

### DISCUSSION THREAD

**Discussion by DearGaz** [umijs/umi#10932] [Q&A]
### build之后部署到服务器上，页面空白，console里没有报错
![image](https://user-images.githubusercontent.com/41284640/230865966-4e62a580-7165-4899-aaf7-1c9fe39c63f8.png)
### dva model的subscriptions添加了路由监听，但是控制台并没有任何输出
![image](https://user-images.githubusercontent.com/41284640/230866350-1b94ff69-698b-478a-b197-14ee1adb8a6e.png)
### 服务器部署目录是/test/pages/，配置的base也是/test/pages/
![image](https://user-images.githubusercontent.com/41284640/230866112-5b6bdb8f-bad4-49c3-88e8-0631945a8c74.png)
### 打包后的静态资源内容看起来正常
![image](https://user-images.githubusercontent.com/41284640/230870063-47608bfd-8184-40b4-9ad0-8fd04ef363af.png)
![image](https://user-images.githubusercontent.com/41284640/230867825-af94942a-e2d4-4473-a73d-dd40d4d3885a.png)

## Context

- **Umi Version**:3.5.39
- **Node Version**:v14.17.5
- **Platform**:Windows 10

**Comment by DearGaz** [comment]
有人吗，比较急，在线等

**Comment by fz6m** [comment]
See #10931
